extensions/mssql/src/objectManagement/ui/databaseDialog.ts

```diff
@@ -1179,6 +1186,14 @@ export class DatabaseDialog extends ObjectManagementDialogBase<Database, Databas
 
 		return data;
 	}
+
+	/**
+	 * Gets the server filestream access level
+	 * @returns true if the server filestream access level is not null and not disabled
+	 */
+	private get serverFilestreamAccessLevel(): boolean {
```

Comment:
Since this is just a boolean value, could the name be changed to something like "serverFilestreamEnabled"?